### PR TITLE
Add environment details to the store UI

### PR DESCRIPTION
**Pull request by Krishanx92**
image3c
image1s

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx [hunk 1]
```diff
@@ -651,126 +646,99 @@ class InfoBar extends React.Component {
...
                                                     <TableCell>21 May 2018</TableCell>
                                                 </TableRow>
-                                                <TableRow>
-                                                    <TableCell component='th' scope='row'>
+                                            </TableBody>
+                                        </Table>
+                                    </div>
+                                </div>
+                                <div className={classes.infoContent}>
+                                    <Typography variant='subtitle2' >
```
Comment (menakaj) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx]: Styling issue.

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx [hunk 2]
```diff
@@ -651,126 +646,99 @@ class InfoBar extends React.Component {
...
+                                        Available Environments</Typography>
+                                    <Grid container spacing={16} item xs={8}>
+                                        {api.endpointURLs.map(i => {
```
Comment (menakaj) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx]: Shall we use a meaningful name for variable 'i'?

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx [hunk 3]
```diff
@@ -651,126 +646,99 @@ class InfoBar extends React.Component {
...
+                                                            {i.environmentType === 'hybrid' && (
+                                                                <Cloud className={classes.iconEven} />
+                                                            )}
+                                                            {i.environmentType === 'production' && (
+                                                                <CheckCircle className={classes.iconEven} />
+                                                            )}
+                                                            {i.environmentType === 'sandbox' && (
+                                                                <Build className={classes.iconEven} />
+                                                            )}<span className={classes.iconTextWrapper}>
+                                                                <Typography className={classes.heading}>{i.environmentName} </Typography></span>
```
Comment (menakaj) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx]: Seems like this line exceeds 120 limit.

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx [hunk 4]
```diff
@@ -651,126 +646,99 @@ class InfoBar extends React.Component {
...
-                                                        <div className={classes.iconAligner}>
-                                                            <LinkIcon className={classes.iconEven} />
-                                                            <span className={classes.iconTextWrapper}>
-                                                                <FormattedMessage
-                                                                    id='Apis.Details.InfoBar.sandbox.url'
-                                                                    defaultMessage='Sandbox URL'
+                                                    </ExpansionPanelSummary>
+                                                    <ExpansionPanelDetails>
+                                                        <Grid container item xs={12} spacing={16}>
+                                                            <Typography className={classes.heading}>Gateway URLs</Typography>
```
Comment (menakaj) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx]: Add i18n for 'Gateway URLs'.
Check the other places and add those as well.

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx [hunk 5]
```diff
@@ -651,126 +646,99 @@ class InfoBar extends React.Component {
...
-                                                                size='small'
-                                                                color='primary'
-                                                                className={classes.margin}
-                                                            >
-                                                                <FormattedMessage
-                                                                    id='Apis.Details.InfoBar.test.endpoint'
-                                                                    defaultMessage='Test Endpoint'
+                                                                    <CopyToClipboard
+                                                                        text={epUrl}
+                                                                        onCopy={this.onCopy('prodUrlCopied')}
```
Comment (menakaj) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx]: This method will automatically call in each render. 
use 
onCopy={() => this.onCopy('prodUrlCopied')} instead.

**Comment by Krishanx92** [comment]
Fix: https://github.com/wso2/product-apim/issues/5003

**Pull request merged**